src/coreclr/debug/ee/debugger.cpp

```diff
@@ -1939,7 +1939,8 @@ HRESULT Debugger::Startup(void)
         if (FAILED(hr))
         {
             ShutdownTransport();
-            ThrowHR(hr);
+            STRESS_LOG0(LF_CORDB, LL_ERROR, "D::S: The debugger pipe failed to initialize in /tmp or $TMPDIR.\n");
+            return S_OK; // we do not want debugger IPC to block runtime initialization
```

Comment:
We need to track properly cleaning up (returning S_FALSE here and if the env var is set so the caller does the cleanup) for .NET 9.

Reply:
I'll add returning S_FALSE.  As Mike points out this will ensure proper cleanup:
https://github.com/tommcdon/runtime/blob/c9b4d5e7ad45de9ef4181cac45439cdc8d60033c/src/coreclr/vm/ceemain.cpp#L1833-L1843

Reply:
I reverted the change and will add the S_FALSE implementation in .NET 9